docs/wiki/pectra-faq.md

```diff
@@ -32,21 +32,38 @@ Overall
 #### **Q:** What changes about deposits?
 #### **Q:** How long do I have to wait for deposits now?
 #### **Q:** What are `0x02` withdrawal credentials?
+Withdrawal credentials with `0x02` prefix means it is a compounding credential. Its associated validator will have its max effective balance increased from 32ETH to 2048ETH and immune to partial withdrawal sweep unless the balance goes over 2048ETH. The word compounding comes from the idea the validator is earning compound interest (up to 2048ETH) since the reward one earns depends on its effective balance. Validators in Deneb are earning simple interest as any amount over 32ETH will be automatically withdrawn.
 #### **Q:** How do I switch to `0x02` withdrawal credentials? How does it help me?
+Only `0x01` validators can switch to `0x02`. There are two ways:
+1) Send a transaction to consolidation request address (pending address to be finalized when Electra goes live on mainnet) with `source_pubkey == target_pubkey`.
```

Comment:
Just to be a bit more practical here, I would specifically mention which tools allow this, e.g.is this going to be part of `ethdo` or staking deposit cli..?